packages/salesforcedx-vscode-core/src/commands/forceLightningAppCreate.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
+import { nls } from '../messages';
+import { notificationService } from '../notifications';
+import { CancellableStatusBar, taskViewService } from '../statuses';
+import {
+  CompositeParametersGatherer,
+  ContinueResponse,
+  DirFileNameSelection,
+  SelectDirPath,
+  SelectFileName,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+const LIGHTNING_APP_EXTENSION = '.app';
+
+class ForceLightningAppCreateExecutor extends SfdxCommandletExecutor<
+  DirFileNameSelection
+> {
+  public build(data: DirFileNameSelection): Command {
```

Comment:
The CLI command will return an error message if it's not in an 'aura' folder. In the user's case, they will see that an error message saying the command failed and in the channel, they can see the error message saying it needs to be in an 'aura' folder